modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/resources/META-INF/frontend-data-set.tld

```diff
@@ -10,6 +10,130 @@
 	<tlib-version>1.0</tlib-version>
 	<short-name>frontend-data-set</short-name>
 	<uri>http://liferay.com/tld/frontend-data-set</uri>
+	<tag>
+		<name>data-set-display</name>
+		<tag-class>com.liferay.frontend.data.set.taglib.servlet.taglib.DataSetDisplayTag</tag-class>
```

Comment:
Oh, you're migrating the remaining tag as well! :open_mouth:  Awesome!

My plan was to name the tag `<frontend-data-set:classic-display>`. I think it's a bit awkward to have double `data-set`: `<frontend-data-set:data-set-display>`, and we want to distance ourselves from `data-set-display` phrase. That should be left for the old component, to prevent confusion in discussions. Along with the change in this file, this would mean:
- tag class should be `ClassicDisplayTag.java`
- jsp dir should be `classic_display`
- attributes should be `frontend-data-set:classic-display:*`

Of course, `classic-display` is not set in stone, but it should be a name that rhymes with `headless-display`.

Reply:
Honestly, I did not have any suggestions for the name. I gave a new name to the tag and waited if the name made sense to you and Brian 😅 

So, I applied the new names that you suggest to this tag in the new commits 🙂

Reply:
Hey guys, just leaving my two cents here. I think we shouldn't mix moving all this with renaming stuff since it would make it more difficult to understand the migration when happens. AFAIK the goal of this first phase is just to move it, not improving nor refactor. Once is done we can talk about what we want to have in the end (which will probably be just one tag).

One step after another :)

Reply:
@carloslancha I agree with you that we should avoid making any improvements unrelated to migration, as a part of the migration. See for example https://github.com/liferay-frontend/liferay-portal/pull/1766#discussion_r771527102

Having said that, rebuilding the component into a set of new modules is the key part of the migration. When we started on this, we could have made a choice to split migration into (1) copy and rename and (2) rebuild into set of modules. It was my call to skip the first step, for better or worse. It would maybe made migration easier for us on technical level, but longer. I figured we can handle this, and I wanted the code in 'final' form to iterate with Brian to the real final form. 

The toughest part in migration will be updates in product teams, and that will be the same unrelated to either approach.